bindings/python/dlite-jstore-python.i

```diff
@@ -0,0 +1,262 @@
+/* -*- Python -*-  (not really, but good for syntax highlighting) */
+
+/* Python-spesific extensions to dlite-storage.i */
+
+%pythoncode %{
+import json
+from typing import Mapping, Sequence
+
+
+def format_dict(
+    d, id=None, soft7=True, single=None, with_uuid=None, with_meta=False,
+    with_parent=True, urikey=False,
+):
+    """Return a copy of `d` formatted according to the given options.
```

Comment:
d should be a datamodel in dict-representation, right? We can't just give it any dict?

Reply:
Yes, you are right. It should be a dict-representation of a datamodel.